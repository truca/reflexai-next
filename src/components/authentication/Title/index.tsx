interface TitleProps {
  title: string;
}

export default function Title({ title }: TitleProps) {
  return (
    <h1 className="text-xl md:text-2xl font-bold leading-tight mt-12">
      {title}
    </h1>
  );
}
