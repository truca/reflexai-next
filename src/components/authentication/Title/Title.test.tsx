import Title from "./index";
import renderer from "react-test-renderer";

describe("Title", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Title title="title" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
