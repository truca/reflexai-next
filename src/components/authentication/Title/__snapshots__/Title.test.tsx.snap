// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title renders correctly 1`] = `
<h1
  className="text-xl md:text-2xl font-bold leading-tight mt-12"
>
  title
</h1>
`;
